packages/pds/src/api/com/atproto/repo.ts

```diff
@@ -241,4 +244,70 @@ export default function (server: Server, ctx: AppContext) {
       })
     },
   })
+
+  server.com.atproto.repo.report({
```

Comment:
bikeshed, but it feels wrong for this to be in the same namespace as repo crud/describe

thoughts on `com.atproto.report.create` or similar?

Reply:
My thought process here was that `com.atproto.repo` was where actions surrounding repos and records would go (i.e. "get a record / create a record / delete a record / report a record")— so not really considering repo mutations to be a special carve-out, but instead just some other actions one can take on a record.

At the same time, I totally understand the tendency to want to separate out the methods that treat the repo concretely as a data structure, rather than abstractly as just another thing the PDS can talk about.

Long way of saying, `com.atproto.report.create` sounds good to me 😎

Reply:
> I totally understand the tendency to want to separate out the methods that treat the repo concretely as a data structure, rather than abstractly as just another thing the PDS can talk about

yup great phrasing of the friction i felt^^ 

I totally see your train of thought here, but I imagine we'll have many more methods that deal with repos/records in some capacity which will bloat that namespace. Seems cleaner to have `com.atproto.repo` methods be for only datastructure operations